app/screens/login.tsx

```diff
@@ -66,12 +72,41 @@ export const LoginScreen: React.FunctionComponent<LoginProps> = observer(props =
     )
   }
 
+	const renderLoginModal = (): React.ReactElement => {
+		return (
+			<View>
+				{/* <TouchableOpacity onPress={toggleAlert}>
+					<Text>OK</Text>
+				</TouchableOpacity> */}
+				<FancyAlert visible={visible} 
+							icon={<View style={styles.loginModalIcon}><Text style={{color : color.palette.white}}>X</Text></View>}
+							style={{ backgroundColor: color.palette.white }}
+							onRequestClose={() => toggleAlert()}
+				>
+					<Text style={styles.loginModalMsg}>An error occurred when logging in please try again later</Text>
+					{/* <TouchableHighlight
+						style={styles.loginModalButton}
+						onPress={() => {
+							toggleAlert();
+						}}
+						>
+						<Text>OK</Text>
+					</TouchableHighlight> */}
+					<TouchableOpacity onPress={toggleAlert}>
+					<Text>OK</Text>
+				</TouchableOpacity>
+				</FancyAlert>
+			</View>
+		);
+	}
+
   return (
```

Comment:
i believe that this kind of modal should be placed more globally so it is reusable.

Reply:
yes i think too! i was not sure, i also want to make a bit order in the ode, i don't know what is the best practice in react-native, i mean i would like to separate the styles in another file, like we do in angular, what is the best practice in react-native ??

Reply:
done , i separated the modal and the loading screen into two separated components, that i could customize when i will get the new UI